### PR TITLE
Fix:ecommerce lookml

### PR DESCRIPTION
**Pull request by axelazaid**
* update `.lkml` files to include measures and dimensions in the ecommerce package
  -  added descriptions to dimensions and measures

### CODE REVIEW

**Comment by jthandy** [comment]
@axelazaid I actually have essentially no background with this package so I'll let @eogilvy12 do the review here!

File: lookml/customers.view.lkml
Comment (eogilvy12) [inline lookml/customers.view.lkml]: I think you should have this field be `created` to keep consistent with removing `date` from your timestamps

File: lookml/customers.view.lkml
Comment (eogilvy12) [inline lookml/customers.view.lkml]: remove first name, last name and phone

File: lookml/ecommerce.model.lkml
Comment (eogilvy12) [inline lookml/ecommerce.model.lkml]: remove label. this is for the ecom package and will not always be shopify data!

File: lookml/ecommerce.model.lkml
Comment (eogilvy12) [inline lookml/ecommerce.model.lkml]: remove shopify

File: lookml/order_items.view.lkml
Comment (eogilvy12) [inline lookml/order_items.view.lkml]: i think this should be a number

File: lookml/order_items.view.lkml
Comment (eogilvy12) [inline lookml/order_items.view.lkml]: i think this should also be a number normally.

File: lookml/order_items.view.lkml
Comment (eogilvy12) [inline lookml/order_items.view.lkml]: this is more frequently called sku not product_sku

File: lookml/orders.view.lkml
Comment (eogilvy12) [inline lookml/orders.view.lkml]: i think this is normally a number as well

File: lookml/orders.view.lkml
Comment (eogilvy12) [inline lookml/orders.view.lkml]: i think we may want to rename this created

File: lookml/orders.view.lkml
Comment (eogilvy12) [inline lookml/orders.view.lkml]: update this description- this is very specific to shopify. it should say this returns 1 if the parameters for a completed order have been fulfilled

File: lookml/orders.view.lkml
Comment (eogilvy12) [inline lookml/orders.view.lkml]: I think this is a shopify field. please check on this. order_seq_number is hidden above but it is what we create as the order number

File: lookml/orders.view.lkml
Comment (eogilvy12) [inline lookml/orders.view.lkml]: I think you should remove billing shipping and location id - these are shopify specific

File: lookml/products.view.lkml
Comment (eogilvy12) [inline lookml/products.view.lkml]: I don't think we need a measure on this table

File: lookml/products.view.lkml
Comment (eogilvy12) [inline lookml/products.view.lkml]: vendor is specific to shopify

**Review by eogilvy12 — CHANGES_REQUESTED** [review]
@alexa I put comments within each file. let me know if you have questions!

once we have this updated, we need to update the dashboard lookml file and test that these changes are maintained before merging in

**Review by eogilvy12 — APPROVED** [review]
this looks excellent! now we can move onto the dashboard lookml updating. I think we should include that in this PR so do not merge this in yet - let's chat about how to best tackle the next step.

**Review by eogilvy12 — CHANGES_REQUESTED** [review]
Can you schedule 15 minutes to review the output of this with me? @axelazaid I think we should add a few reports to the main dashboard

**Comment by clrcrl** [comment]
Closing due to deprecation

**Pull request closed without merging**
